website/content/docs/concepts/events.mdx

```diff
@@ -30,39 +30,39 @@ additional `metadata` field.
-| database | `database/role-update`              | `modified`, `operation`, `path`, `name`        | 1.16          |
-| database | `database/root-rotate-fail`         | `modified`, `operation`, `path`, `name`        | 1.16          |
-| database | `database/root-rotate`              | `modified`, `operation`, `path`, `name`        | 1.16          |
-| database | `database/rotate-fail`              | `modified`, `operation`, `path`, `name`        | 1.16          |
-| database | `database/rotate`                   | `modified`, `operation`, `path`, `name`        | 1.16          |
-| database | `database/static-creds-create-fail` | `modified`, `operation`, `path`, `name`        | 1.16          |
-| database | `database/static-creds-create`      | `modified`, `operation`, `path`, `name`        | 1.16          |
-| database | `database/static-role-create`       | `modified`, `operation`, `path`, `name`        | 1.16          |
-| database | `database/static-role-delete`       | `modified`, `operation`, `path`, `name`        | 1.16          |
-| database | `database/static-role-update`       | `modified`, `operation`, `path`, `name`        | 1.16          |
-| kv       | `kv-v1/delete`                      | `modified`, `operation`, `path`                | 1.13          |
-| kv       | `kv-v1/write`                       | `data_path`, `modified`, `operation`, `path`   | 1.13          |
-| kv       | `kv-v2/config-write`                | `data_path`, `modified`, `operation`, `path`   | 1.13          |
-| kv       | `kv-v2/data-delete`                 | `modified`, `operation`, `path`                | 1.13          |
-| kv       | `kv-v2/data-patch`                  | `data_path`, `modified`, `operation`, `path`   | 1.13          |
-| kv       | `kv-v2/data-write`                  | `data_path`, `modified`, `operation`, `path`   | 1.13          |
-| kv       | `kv-v2/delete`                      | `modified`, `operation`, `path`                | 1.13          |
-| kv       | `kv-v2/destroy`                     | `modified`, `operation`, `path`                | 1.13          |
-| kv       | `kv-v2/metadata-delete`             | `modified`, `operation`, `path`                | 1.13          |
-| kv       | `kv-v2/metadata-patch`              | `data_path`, `modified`, `operation`, `path`   | 1.13          |
-| kv       | `kv-v2/metadata-write`              | `data_path`, `modified`, `operation`, `path`   | 1.13          |
-| kv       | `kv-v2/undelete`                    | `data_path`, `modified`, `operation`, `path`   | 1.13          |
+| Plugin   | Event Type                          | Metadata                                                      | Vault version |
+|----------|-------------------------------------|---------------------------------------------------------------|---------------|
+| database | `database/config-delete`            | `modified`, `operation`, `path`, `name`, `vault_index`        | 1.16          |
```

Comment:
Is `vault_index` available in 1.16?

Reply:
Good question! `vault_index` is only available in 1.20. My thought was to let versioned docs handle this, i.e. the 1.16 version will not list `vault_index`, but the 1.20 version will.